clcache.py

```diff
@@ -1530,8 +1530,11 @@ def main():
         print('Statistics reset')
         return 0
 
-    if len(sys.argv) == 3 and sys.argv[1] == "-M":
-        arg = sys.argv[2]
+    if (len(sys.argv) == 3 and sys.argv[1] == "-M") or "CLCACHE_MAXSIZE" in os.environ:
+        if "CLCACHE_MAXSIZE" in os.environ:
+            arg = os.environ["CLCACHE_MAXSIZE"]
+        else:
+            arg = sys.argv[2]
```

Comment:
I find it strange that the environment variable overrides the command line argument. I think it's more common to have it the other way round, isn't it?